### ISSUE TITLE
v1.1 discussion. See zip file below for v1.1

### ISSUE THREAD

**Issue by jackneil** [hank-ai/hank-ai.github.io#7]
https://drive.google.com/file/d/1o_JmpyJNX75NQqFFjK5t2eCu9Q6KAPwE/view?usp=sharing

This isn't the best way to do this discussion but wanted to get it started in an open trackable centralized location ... so here we are.

Also, I am totally aware that many of my thoughts in this post are not novel and are probably already in the plans of Sam, Travis, and Hasani. So please don't feel insulted by any brevity or minutia I include here. Just wanting to be thorough.

Overall theme: 
1. Needs help. I like the small top and bottom 'frames' and the gray-white-black color scheme
2. Don't like the way the top links look though. Looks a little too simple for a high-tech company. At least hover-overs or some other robustness. Probably tighter together as well? 
3. Bottom panel: too empty. The copyright should have much less focus (don't want legal crap to be so big on our landing page). Probably put social media links there too to the main company ones (linkedin, twitter - if we had one). 
4. Should everything on all pages be center-aligned? Or is left-aligned for text more normal? 
5. I'd put the large icon and slogan on the home page but on all other pages I think I'd move the icon into the top-left corner title-bar and move the slogan to the bottom 'frame'. Don't think we need the slogan at the top of all pages. But I could be won-over on this if everyone disagrees with me.
6. More pictures?

index: 
1. Like the rotating quotes (need more, and need randomization, of course). 
2. Like the quote on that page about our committment to responsibility. Maybe tweak it slightly to "Hank is committed to developing responsible artificial intelligence."? Is that better grammar or worse? I dunno. Not an english major. May be better like it is currently.

about1: 
1. Overall, good concept and content. Needs a little tweaking and the last paragraph needs some TLC.
2. Hasani is working on the full-story for hank.ai so he may have a lot of feedback/additions to this section.

arch: Great. Probably want to pull some of the details out of our web-presence and keep it for investor prospectus though. Keep the water murky, not too clear.

team: 
1. Great layout. Love the panels. 
2. Maybe include the entire team, but we need to discuss that as a group. 
3. I'd make the link targets a '_blank' in the href. 
4. Also might want to change that one big 'see profile' link into links to our social media and code accounts (linkedin, github, twitter, myspace, etc)

contact link: let's point it to info@hank.ai instead of contact@hank.ai 

Thanks folks!

**Comment by shartzog** [comment]
thanks, jack. i wasn't too pleased with it overall, but thought it was a start.  my main issue for now is not knowing what to add.  any ideas for flashy content would be welcome.  also, all totally ootb bootstrap atm.  could add scripting but didn't know anything i wanted to script... :)

also, was planning to push source to a new branch but doesn't look like i have access to do so.  either way, here were comments I was planning to include with the new branch:

I'm really not happy with this but ran out of steam in terms of content. Outstanding items:
1) team page in general. pics, descriptions, which soc med links, etc.
2) too much text. tells the story well but much too wordy.
3) is this the right audience? cater to investors or potential users?
4) not enough "flare". how can we make it more exciting?
5) not friendly w IE9 & earlier.  is this a problem?
6) consolidate story pages into a single page with different sections or a carousel with all of the content?

responses to comments above...
overall theme:
1) agree on both points.  let me know what you like.  spent some time looking over lists of "top X startup websites".  there's plenty of diversity but wasn't sure what to do to make ours more flashy.  ideas welcome...
2) also agree.  spent zero time on this so far.  spacing looks good on phone, not on pc, but needs to be "flashier" either way.
3) I cut the slogan out of it due to mobile considerations...  that's literally all that will fit in the bottom pane unless we stack some things for mobile users.
4) If we stick with this much text, i think it has to be left aligned or justified for readability.  but also not thrilled with this much text.
5) agree with this, but have to replace w something.  ideas?
6) Definitely.  of what tho?

index:
1) on the fence on both points.  quotes are all tied to specific themes that are deep dived in the "about" pages.  that said, i read tons of quotes that i thought could fit the bill.  maybe have randomized quotes across all three themes on main page and show a carousel of quotes applying to each theme on story pages?  for reference, themes i was going for: (a) ai is coming whether you like it or not (b) there will be negative impacts and hank is aware and here to help and (c) "hank isn't human. so you can be"...
2) like as is.  a bit of a dangling modifier with "responsibly" but it's an extremely common phrasing at this point.  open to changing though.

about:
1) still think the whole business is too wordy.  overall, i've cut out about half of what i'd orginally written, but not enough... lol
2) Help, Hasani!!! :)

team:
didn't spend much time here yet.  wanted to get feedback from team on who to include and overall tone.  also just stole slack pics for now.  will need real pics and an idea of which soc med links to include. (will make them branded of course).

**Comment by beeryardtech** [comment]
<http://beeryardtech.com/hank-ai.github.io/>

Created a fork so can view online the changes, once we get what we want we can talk about laying things out the "jekyll" way. 

Supported Browsers: **Chrome, Firefox, Edge**. Mention IE, will shit a brick.

**Comment by jackneil** [comment]
Thanks Travis. Assume Safari is in that list too?

Sent from my iPhone. Typo forgiveness requested.

Jack Neil, MD
(m) +1.803.466.7551
linkedin.com/in/jackneil
@realjackneil
